docs/deployment/settings.md

```diff
@@ -1,5 +1,22 @@
 <!-- DO NOT MODIFY THIS FILE DIRECTLY, IT IS AUTO GENERATED BY [org.apache.kyuubi.config.AllKyuubiConfiguration] -->
```

Comment:
Can we add the license header before this line?

Reply:
Updated.

Reply:
hmm.. I mean "**before**" this line.

Reply:
Oops, missing this file ...